CITATION.cff

```diff
@@ -0,0 +1,25 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: pixi
+message: >-
+  If pixi contributes to a project that leads to a
+  scientific publication, please acknowledge this fact by
+  citing according to this file.
+type: software
+authors:
+  - given-names: pixi open-source contributors
+repository-code: 'https://github.com/prefix-dev/pixi'
+url: 'https://pixi.sh'
```

Comment:
As you have versioned docs (e.g. https://pixi.sh/v0.15.2/) I would include this information here as well

```
url: "https://pixi.sh/v0.15.2/
```